react/scss/timeline-app.scss

```diff
@@ -0,0 +1,76 @@
+@import 'common';
+
+.timeline {
+  .vis-item {
+    background-color: #eee;
+    border-color: #444;
+  }
+
+  .vis-tooltip {
+    display: none;
+  }
+
+  &__itembox {
```

Comment:
usually, it is a good convention to only nest modifiers (button--large would nest --large) but each element should get its own block (for button__icon __icon won't be nested). This leads to more consistent code that is easier to read. Not required to change anything just for info.